ReactWindows/ReactNative.Shared/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,229 @@
+using ReactNative.UIManager;
```

Comment:
>u [](start = 0, length = 1)

What's the plan for handling the case where new elements get inserted into the tree after `importantForAccessibility` has already been set higher in the tree?

Reply:
I introduced a method InitImportantForAccessibility to be called for newly inserted elements. It will set the AccessibilityView to Raw if it finds that an ancestor "hides" the newly inserted element. That is the only case when setting is needed upon initial insertion.


---
In reply to: [163134701](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r163134701) [](ancestors = 163134701)